packages/angular_devkit/core/src/terminal/index.ts

```diff
@@ -6,4 +6,5 @@
  * found in the LICENSE file at https://angular.io/license
  */
 export * from './text';
+export * from './caps';
```

Comment:
This doesn't appear to be used anywhere?

Reply:
It is in the architect CLI.

Reply:
(see the progress bar stuff).